### DISCUSSION TITLE
Steam Big Picture Mode Beta

### DISCUSSION THREAD

**Discussion by ChadhaAnuj** [Belphemur/SoundSwitch#1083] [FAQ]
Amazing application. The interface is wonderful and it works perfectly to change audio for steam big picture mode. However, it does not seem to work when I try to use it for the new big picture mode beta. Is this because it is the beta and not the official? Or will this require some update to see it. I am just curious for the future when that update becomes the normal.

**Comment by Belphemur** [comment]
Hello,

Please see this issue:
https://github.com/Belphemur/SoundSwitch/issues/1074

I haven't had the time to fix the problem with the new overlay of steam.
